### PR TITLE
Master

### PR DESCRIPTION
**Pull request by fduarte42**
Added JsonServer example

### CODE REVIEW

File: Classes/Service/JsonServer.php [hunk 1]
```diff
@@ -118,7 +118,7 @@ protected function getServiceUri()
     {
         $uri = GeneralUtility::getIndpEnv('TYPO3_REQUEST_URL');
         $parts = parse_url($uri);
-        $parts['query'] = 'eID=JsonServer&amp;server=' . $this->serverKey;
+        $parts['query'] = 'eID=JsonServer&server=' . $this->serverKey;
```
Comment (lochmueller) [inline Classes/Service/JsonServer.php]: Why you reset this change: https://github.com/lochmueller/autoloader/pull/59
?

Comment (lochmueller) [inline Classes/Service/JsonServer.php]: I saw you explicit set this: https://github.com/lochmueller/autoloader/pull/65/commits/fed22a5e17d309d79a6b8681a55eaa2d513e8ec8

So I merge your example extension.... thanks

Comment (fduarte42) [inline Classes/Service/JsonServer.php]: Add changed it in the SoapServer, but in the JsonServer it is wrong.

So SoapServer needs encoded Ampersand, but JsonServer does not.

Comment (lochmueller) [inline Classes/Service/JsonServer.php]: Ahh... thanks for your contribution. Patch is merged.

**Pull request merged**